Makefile

```diff
@@ -9,14 +9,12 @@ clean:
 
 format:
 	(./code/gradlew -p code/$(EXTENSION-LIBRARY-FOLDER-NAME) spotlessApply)
-	(./code/gradlew -p code/$(TEST-APP-FOLDER-NAME) spotlessApply)
 		
 format-license:
 	(./code/gradlew -p code licenseFormat)
 
 checkformat:
 	(./code/gradlew -p code/$(EXTENSION-LIBRARY-FOLDER-NAME) spotlessCheck)
-	(./code/gradlew -p code/$(TEST-APP-FOLDER-NAME) spotlessCheck)
 
```

Comment:
Actual code change. Not just formatting.
Removed applying spotless on test apps.